cmd/create.go

```diff
@@ -186,6 +190,7 @@ func init() {
 	componentCreateCmd.Flags().StringVar(&componentGit, "git", "", "Use a git repository as the source file for the component")
 	componentCreateCmd.Flags().StringVar(&componentLocal, "local", "", "Use local directory as a source file for the component")
 	componentCreateCmd.Flags().StringSliceVar(&componentPorts, "port", []string{}, "Ports to be used when the component is created")
+	componentCreateCmd.Flags().StringSliceVar(&componentEnvVars, "env", []string{}, "Environmental variables for the component")
```

Comment:
Help message should explain how env variables are specified (what is the format)

Reply:
`Environmental variables for the component, specified like --env Key=Value` Is it good enough?

Reply:
It might be better to use something like `VariableName` or something like that indicates that it is name of the variable rather than `Key`